collectors/apps.plugin/apps_plugin.c

```diff
@@ -440,6 +442,7 @@ size_t pagesize;
 #define PID_LOG_CMDLINE 0x00000004
 #define PID_LOG_FDS     0x00000008
 #define PID_LOG_STAT    0x00000010
+#define PID_LOG_UPTIME  0x00000011
```

Comment:
Shoudn't `PID_LOG_UPTIME` be `0x00000020`? All previously defined flags set non overlapping bits. If those are ever ORed or ANDed it would be a problem.

Reply:
I've changed it. Thanks!